presto-spi/src/main/java/com/facebook/presto/spi/StandardErrorCode.java

```diff
@@ -140,6 +141,7 @@ public enum StandardErrorCode
     EXCEEDED_WRITTEN_INTERMEDIATE_BYTES_LIMIT(0x0002_0012, INSUFFICIENT_RESOURCES),
     TOO_MANY_SIDECARS(0x0002_0013, INTERNAL_ERROR),
     NO_CPP_SIDECARS(0x0002_0014, INTERNAL_ERROR),
+    FUNCTION_SERVER_FAILURE(0X0002_0015, EXTERNAL),
```

Comment:
We should use an error code that is specific to the module, it shouldn't be in the SPI.

Reply:
Added RestErrorCode that is specific to the module.